mm2src/mm2_main/tests/docker_tests/docker_tests_common.rs

```diff
@@ -377,6 +379,40 @@ pub fn geth_docker_node<'a>(docker: &'a Cli, ticker: &'static str, port: u16) ->
     }
 }
 
+pub fn sol_asset_docker_node<'a>(docker: &'a Cli, ticker: &'static str) -> DockerNode<'a> {
+    let port = 8899;
+
+    let image = GenericImage::new(SOLANA_CLUSTER_DOCKER_IMAGE, "latest");
+    let image = RunnableImage::from((image, vec![])).with_mapped_port((port, port));
+    let container = docker.run(image);
+
+    let rt = Runtime::new().unwrap();
+    rt.block_on(async {
+        tokio::time::sleep(Duration::from_secs(30)).await;
+    });
+    let container_id = container.id().to_string();
+    let output = Command::new("docker")
+        .arg("exec")
+        .arg(&container_id)
+        .arg("/bin/sh")
+        .arg("-c")
+        .arg("cd /root && ./mint.sh")
+        .output()
+        .expect("Failed to execute script in the container");
```

Comment:
Can't this be part of the container entrypoint?

Reply:
@onur-ozkan I need the node to start running before I can execute this script. The solana-test-validator is the Docker's entry point.